Author: Andreas Tille <tille@debian.org>
last-Update: Thu, 31 Aug 2017 23:24:39 +0200
Bug-Debian: https://bugs.debian.org/853486
Description: Do not set -Werror which makes build fail when using gcc-7

--- libcereal.orig/CMakeLists.txt
+++ libcereal/CMakeLists.txt
@@ -18,10 +18,6 @@
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj /W3 /WX")
 else()
     set(CMAKE_CXX_FLAGS "-Wall -g -Wextra -Wshadow -pedantic -Wold-style-cast ${CMAKE_CXX_FLAGS}")
-    option(WITH_WERROR "Compile with '-Werror' C++ compiler flag" ON)
-    if(WITH_WERROR)
-        set(CMAKE_CXX_FLAGS "-Werror ${CMAKE_CXX_FLAGS}")
-    endif(WITH_WERROR)
     if(CMAKE_VERSION VERSION_LESS 3.1)
         set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
     else()
